tests/unit/getters/get-tx-amount-total.spec.js

```diff
@@ -1,4 +1,5 @@
 import BigNumber from 'bignumber.js';
+import '@/protocols/registerAdapters';
```

Comment:
We can extract duplicate imports in the `setup` file, if it suits the situation. https://github.com/aeternity/superhero-wallet/commit/891c8a3effba0eb6df8098165affd1da42991e13#diff-1e058ca1442e46581b13571fb8d261f9e1f5657e26c96634d4c1072f0f0347f1L22

Reply:
Nice idea. PR aligned.